.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+env:
+  global:
+      - secure: uytPp0Fs+LT3QDEhDM3LJWiLvT2AHbdWnXrPEs+bYshQwt9wST+KQnYLyRfBuGg2ux3pkZwsRUFexvN8pQ3ab4aU2P21Xo98TzdXRwXurNYePgk/3tykEH+JrL52DfjCWB1VsjzzFrP02XU0XtB30qWC/n+fxeMWT7JT2GVh/OE=
```

Comment:
I guess this will have to get regenerated for this repository, since the encryption keys are unique to each repository.